core/src/main/java/bisq/core/payment/payload/UpholdAccountPayload.java

```diff
@@ -85,12 +95,20 @@ public static UpholdAccountPayload fromProto(protobuf.PaymentAccountPayload prot
 
     @Override
     public String getPaymentDetails() {
-        return Res.get(paymentMethodId) + " - " + Res.getWithCol("payment.account") + " " + accountId;
+        return Res.get(paymentMethodId) + " - " + getPaymentDetailsForTradePopup().replace("\n", ", ");
     }
 
     @Override
     public String getPaymentDetailsForTradePopup() {
-        return getPaymentDetails();
+        if (accountOwner.equals("")) {
```

Comment:
`"".equals(accountOwner)` ?

Reply:
I don't think we need the extra null check for accountOwner in this case as it is always initialized with `""`. Probably `accountOwner.isEmpty()` would be more readable for this backwardscomp. check. @BtcContributor WDYT?

Reply:
Thanks, good catch!